src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -1049,18 +1054,18 @@ private static BoundExpression MakeLiteral(SyntaxNode syntax, ConstantValue cons
                     var unusedDiagnostics = DiagnosticBag.GetInstance();
 
                     argument = GetDefaultParameterValue(syntax,
-                        parameterOfOptionalParametersMethod, 
-                        enableCallerInfo: ThreeState.Unknown, 
-                        localRewriter: null, 
-                        binder: binder, 
+                        parameterOfOptionalParametersMethod,
+                        enableCallerInfo: ThreeState.Unknown,
+                        localRewriter: null,
+                        binder: binder,
                         diagnostics: unusedDiagnostics);
                     kind = ArgumentKind.DefaultValue;
 
-                    unusedDiagnostics.Free();                                                                                       
-                }                                                              
+                    unusedDiagnostics.Free();
+                }
 
-                argumentsBuilder.Add(CSharpOperationFactory.CreateArgumentOperation(kind, parameter, CSharpOperationFactory.Create(argument)));
-            }                                                               
+                argumentsBuilder.Add(CSharpOperationFactory.CreateArgumentOperation(kind, parameter, operationFactory.Create(argument)));
```

Comment:
>operationFactory.Create(argument) [](start = 101, length = 33)

Another place where we are probably caching nodes that are not part of the original tree.